reliability-v2/config/reliability.yaml

```diff
@@ -36,13 +36,16 @@ reliability:
         - func apply_nonamespace -p "<path_to_content>/network-policy/02_anp_allow-traffic-egress-cidr-cluster-network-p20.yaml"
         - func apply_nonamespace -p "<path_to_content>/network-policy/04_anp_allow-traffic-dev-test-p30.yaml"
         - func apply_nonamespace -p "<path_to_content>/network-policy/03_anp_allow-traffic-dev-prod-p31.yaml"
+        - <path_to_script>/list_all_resources.sh
```

Comment:
@mehabhalodiya To easily analysis the results (especially by an automation script), I think it is helpful to output the test results to a separate files with timestamp. Or else you have to find out the results in the reliability.log.

Reply:
Done. @qiliRedHat You can refer to the below screenshot.